### PR TITLE
IDNSEIT-887 Remove sailpoint-ui-common and its dependencies

### PR DESCRIPTION
**Pull request by navaneeth-basavaraj-SP**

### CODE REVIEW

File: grid-ui-video/package.json [hunk 1]
```diff
@@ -3,30 +3,48 @@
...
-    "test:debug": "karma start karma.conf.js --debug true"
+    "start": "ng serve",
+    "build": "ng build",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e",
+    "publish": "npm run build && node node_modules/sailpoint-angular-common/build/tasks/publish.js --prod"
   },
-  "private": true,
+  "private": false,
```
Comment (scott-phillips-sp) [inline grid-ui-video/package.json]: umm... this is still a private repo... you should leave this as true.

File: grid-ui-video/slptConfig.json [hunk 2]
```diff
@@ -1,29 +1,17 @@
...
-
 	// The project's repository
 	"repo": {
 		"url": "https://github.com/sailpoint/Grid"
 	},
 
 	// Where to publish builds to
 	"publish": {
-		"bucket": "files.cloud.sailpoint.com",
-		"destPath": "modules/builds/grid/{buildNumber}",
```
Comment (scott-phillips-sp) [inline grid-ui-video/slptConfig.json]: can you remove any old builds that might have gotten published to files.cloud.sailpoint.com/modules?

File: grid-ui-video/src/app/dashboard/dashboard.component.ts [hunk 3]
```diff
@@ -11,6 +11,7 @@ import {Http, Headers} from '@angular/http';
 import {DomSanitizer} from '@angular/platform-browser';
 import {DialogComponent} from "../dialog/dialog.component";
 import {environment} from "../../environments/environment";
+import 'rxjs/Rx';
```
Comment (scott-phillips-sp) [inline grid-ui-video/src/app/dashboard/dashboard.component.ts]: Why?

Comment (navaneeth-basavaraj-SP) [reply grid-ui-video/src/app/dashboard/dashboard.component.ts]: The getVideos method needs that import to map the json response.
this.http.get(environment.serviceURL, {headers: this.getHeaders()})
            .map(response => <Video[]>response.json())

File: grid-ui-video/slptConfig.json (hunk 2 above)
Comment (navaneeth-basavaraj-SP) [reply grid-ui-video/slptConfig.json]: Removed grid module from that s3 bucket.

**Pull request merged**